### DISCUSSION TITLE
Actual field names for export template

### DISCUSSION THREAD

**Discussion by AnkeTurczak** [netbox-community/netbox#7929] [General]
Hi,

I'm still quite new to Netbox and trying to get into it. 
We are currently trying to use an export template to pull certain data out of Netbox. In order to do this, I need to know the actual field name as it appears under the list in the admin UI.
Can anyone tell me how I can get there or where else I could find out what the actual field names are?

Thank you very much.

**Comment by jk-onnet** [comment]
1) You can review the code of netbox. Since the netbox is built in django, each model data is mostly 'app/{model}' 
app refer to dcim, ipam and etc
model refer to device, ip address, and etc
2) api doc
However, if you are using custom field, tags and others, you might need to find how the function is built or the library used.

**Reply by AnkeTurczak** [reply]
Ok, found it now.
Thank you very much.
